email-utils.js

```diff
@@ -3,6 +3,18 @@
 const AppConstants = require("./app-constants");
 
 const nodemailer = require("nodemailer");
+const hbs = require("nodemailer-express-handlebars");
+
+const hbsOptions = {
+  viewEngine: {
+    extname: ".hbs",
+    layoutsDir: __dirname + "/views/layouts/",
+    defaultLayout: "email",
+    partialsDir: __dirname + "/views/partials/",
+  },
+  viewPath: __dirname + "/views/email/",
```

Comment:
Not sure how relevant this is, but this may need a `path.join()` type thing on these `__dirname` references to make it more cross-OS compatible.